quinn-proto/src/connection.rs

```diff
@@ -165,7 +165,7 @@ pub struct SentPacket {
     pub time: u64,
     /// 0 iff ack-only
     pub bytes: u16,
-    pub handshake: bool,
+    pub initial: bool,
```

Comment:
This flag is used to implement the draft-11 handshake retransmit strategy, which is obsolete. I'm not sure renaming it makes sense; it doesn't preserve the original meaning (which included `Handshake` packets) and I'm not sure we need anything like it with the new handshake structure.

Reply:
I'll just keep this as `handshake` for now, then.